cli/pom.xml

```diff
@@ -182,6 +182,13 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration combine.self="override">
```

Comment:
What issue does this `combine.self` solve?

Reply:
Preventing overriding the parent configuration (See https://maven.apache.org/pom.html ) - I think, if I recall correctly, the CLITest was not suitable for forking etc

Reply:
Do you mean the parent from overriding the child configuration? I think there is `<inherited>false</inherited>` to prevent merging the configuration with the child. I see `inherited` used in a lot of the examples in that URL, but nowhere in the text do they discuss that option.

I wasn't familiar with this option before, so a comment would have been helpful to me.

Reply:
Ah ok. Its the inverse of `<inherited>false` as we normally want the inheritance except for this module. Can go over it offline if you want.